vue/official/tabs/src/views/TabsPage.vue

```diff
@@ -4,17 +4,17 @@
       <ion-router-outlet></ion-router-outlet>
       <ion-tab-bar slot="bottom">
         <ion-tab-button tab="tab1" href="/tabs/tab1">
-          <ion-icon :icon="triangle" />
+          <ion-icon :aria-hidden="true" :icon="triangle" />
           <ion-label>Tab 1</ion-label>
         </ion-tab-button>
 
         <ion-tab-button tab="tab2" href="/tabs/tab2">
-          <ion-icon :icon="ellipse" />
+          <ion-icon :aria-hidden="true" :icon="ellipse" />
```

Comment:
```suggestion
          <ion-icon aria-hidden="true" :icon="ellipse" />
```
`aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.